### ISSUE TITLE
Enabling `autoprefixer` on a theme breaks the deploy

### ISSUE THREAD

**Issue by gcmznt** [liferay/liferay-js-themes-toolkit#274]
Enabling `autoprefixer` on Minium theme breaks the deploy just after the postCSS gulp plugin

**Comment by wincent** [comment]
@gcmznt: what does the breakage look like? (ie. anything printed to the console?)

**Comment by gcmznt** [comment]
I had this problem running the deploy task via `gradle`, running the task via `gulp` it works.
With the gradle deploy task the gulp build just fail silently when autoprefixer is piped

**Comment by wincent** [comment]
@gcmznt: I'm having a bit of trouble reproducing this, because I am not sure what version you're working with at the moment.

At least on "master" (which uses v9 of the themes toolkit), I tried turning on autoprefixer in `modules/apps/frontend-theme-fjord/frontend-theme-fjord` (I am not sure where to get this Minium theme that you mentioned).

```diff
diff --git a/modules/apps/frontend-theme-fjord/frontend-theme-fjord/package.json b/modules/apps/frontend-theme-fjord/frontend-theme-fjord/package.json
index e9b18ba4b6f6..538ec5bde76a 100644
--- a/modules/apps/frontend-theme-fjord/frontend-theme-fjord/package.json
+++ b/modules/apps/frontend-theme-fjord/frontend-theme-fjord/package.json
@@ -11,7 +11,10 @@
                "baseTheme": "styled",
                "distName": "fjord-theme",
                "rubySass": false,
-               "version": "7.1"
+               "version": "7.1",
+               "postcss": [
+                       "autoprefixer"
+               ]
        },
        "main": "package.json",
        "name": "liferay-fjord-theme",
```

When I run `gradlew clean deploy` I see the deploy finish without errors and it includes this output suggesting that autoprefixer was invoked:

```
[12:11:11] gulp-postcss: main.css
autoprefixer: /Users/greghurrell/code/portal/liferay-portal/modules/apps/frontend-theme-fjord/frontend-theme-fjord/build/_css/main.css:2568:9: You should write display: flex by final spec instead of display: box
autoprefixer: /Users/greghurrell/code/portal/liferay-portal/modules/apps/frontend-theme-fjord/frontend-theme-fjord/build/_css/main.css:2635:3: You should write display: flex by final spec instead of display: box
autoprefixer: /Users/greghurrell/code/portal/liferay-portal/modules/apps/frontend-theme-fjord/frontend-theme-fjord/build/_css/main.css:2662:5: You should write display: flex by final spec instead of display: box
Done in 4.79s.
```

Can you let me know which version of the portal you're working in, and which version of the themes toolkit? And if you think there is anything specific to the Minium theme that is necessary to repro this, let me know how to get a hold of it? Thanks.

**Comment by wincent** [comment]
Actually, scratch that! I misread the deploy log: it actually says

```
> Task :apps:frontend-theme-fjord:frontend-theme-fjord:deploy NO-SOURCE
```

It's supposed to say something like "Files of project ':apps:frontend-theme-fjord:frontend-theme-fjord' deployed to /Users/greghurrell/code/portal/bundles/deploy" instead of "NO SOURCE".

I can take it from here!

**Comment by wincent** [comment]
Temporary workaround until I figure out how/why `gulp-plumber` is breaking the build; comment out this line: https://github.com/liferay/liferay-js-themes-toolkit/blob/e8400255990e8bdc26f16ce3320f8517ff557572/packages/liferay-theme-tasks/tasks/build/compile-css.js#L71

in `liferay-portal/modules/node_modules/liferay-theme-tasks/tasks/build/compile-css.js`.

**Comment by wincent** [comment]
Just published v8.0.6 and v9.0.0-beta.1 which contain the fix for this.
